packages/semver/src/executors/version/utils/git.ts

```diff
@@ -5,26 +5,52 @@ import { exec } from '../../common/exec';
 import { logStep, _logStep } from './logger';
 
 /**
- * Return the list of commits since `since` commit.
+ * Return the list of commit bodies since `since` commit.
  */
 export function getCommits({
   projectRoot,
   since,
 }: {
   projectRoot: string;
   since: string;
+}): Observable<string[]> {
+  return getFormattedCommits({since, projectRoot, format:'%B'})
+}
+/**
+ * Return hash of last commit of a project
+ */
+export function getLastProjectCommitHash({
```

Comment:
I would rename it to `getCommitHash`, the param is explicit so we don't have to specify it in the function name IMO.